dump/parser.go

```diff
@@ -82,7 +82,7 @@ func Parse(r io.Reader, h ParseHandler, parseBinlogPos bool) error {
 				return errors.Errorf("parse values %v err", line)
 			}
 
-			if err = h.Data(db, table, values); err != nil && err != ErrSkip {
```

Comment:
em, any reason to raise ErrSkip?

Reply:
I did it because Data function (in the same file) swallows any parse errors and returns  ErrSkip instead. Practically in my case the whole table was silently skipped during mysqldump. That table had two binary columns and at least one of them was empty.

Probably the proper fix would be to return original parse error from Data function instead of ErrSkip

Reply:
No problem. I'll do it later today

Reply:
I've just added new commit. Dump parser now returns original errors. 
Also I reverted `err != ErrSkip` check even though there's not code that returns that error